src/Microsoft.Azure.SignalR/ServerConnections/ClientConnectionContext.cs

```diff
@@ -37,7 +37,9 @@ internal class ClientConnectionContext : ConnectionContext,
 
         private readonly TaskCompletionSource<object> _connectionEndTcs = new TaskCompletionSource<object>(TaskCreationOptions.RunContinuationsAsynchronously);
 
-        public Task CompleteTask => _connectionEndTcs.Task;
+        public Task CompleteTask => _connectionEndTcs.Task;
+
+        public readonly bool IsMigrated;

```

Comment:
>IsMigrated [](start = 29, length = 10)

as a convention, use property instead of field for public ones 
`public bool IsMigrated {get;}`